### PR TITLE
(DPS-2121) Clean up Changelog v3.2

### PR DESCRIPTION
**Pull request by ale-kacperczyk**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/dps-2121)
- [Jira](https://livechatinc.atlassian.net/browse/DPS-2121)

## 📓 Description
Restructured the Changelog in Agent Chat API as a proof of concept.

### Content

- Proofreading/content fixes

### CODE REVIEW

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 1]
```diff
@@ -11,64 +11,86 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
...
-- There's a new method **List Threads** ([Web](/messaging/agent-chat-api/v3.2/#list-threads) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-threads)).
 - There's a new `highlights` parameter in the **List Archives** ([Web](/messaging/agent-chat-api/v3.2/#list-archives) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-archives)) method. It allows you to highlight the match of `filters.query` with an HTML tag.
-- There's a new push [`queue_positions_updated`](/messaging/agent-chat-api/v3.2/rtm-reference/#queue_positions_updated) with updated positions and wait times for queued chats.
 
-### Changed
+- The **Get Chats Summary** method was renamed to **List Chats**.
+- There's a new `include_chats_without_threads` filter in the  **List Chats** ([Web](/messaging/agent-chat-api/v3.2/#list-chats) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-chats)) method.
+- The `order` field was renamed to `sort_order` in **List Chats**.
+- The `order` field in the `chat` object was removed from the responses of the **List Chats** method.
+
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/changelog/index.mdx:26]: I have these empty lines to help me group changes, but I will remove them in the final version.

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 2]
```diff
@@ -11,64 +11,86 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
...
+- The **Get Chat Threads Summary** method was removed.
+
+- There's a new push, [**chat_unfollowed**](/messaging/agent-chat-api/v3.2/rtm-reference/#chat_unfollowed), complementary to the **Unfollow Chat** method.
+
+### Chat access
 
-- The **Update Agent** method was replaced with **Set Routing Status** ([Web](/messaging/agent-chat-api/v3.2/#set-routing-status) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#set-routing-status)).
-- The `agent_updated` push was replaced with [`routing_status_set`](/messaging/agent-chat-api/v3.2/rtm-reference/#routing_status_set).
-- The method **Update Customer** ([Web](/messaging/agent-chat-api/v3.2/#update-customer) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#update-customer)), which used to return the Customer data structure, now has no response payload.
-- The method **Get Chat Threads** ([Web](/messaging/agent-chat-api/v3.2/#get-chat-threads) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-chat-threads)) now requires `thread_ids`. Also, it no longer returns `threads_summary` in the response.
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/changelog/index.mdx:32]: This was probably leftover. I removed it completely.

**Comment by ale-kacperczyk** [comment]
@bkubiak @gluszczyk WDYT? Should I proceed with the other Changelogs?

**Comment by gluszczyk** [comment]
> @bkubiak @gluszczyk WDYT? Should I proceed with the other Changelogs?

Looks promising. Few things after a quick look.

I like it how renames are followed by changes to the method/push under the new name.

Maybe we could group together changes spread across multiple points? For example, `ListArchives` has 4 separate changes listed that could be moved to a nested list?

Moving method/push names as much to the left will also make it easier to go through the changelog quickly. If I'm looking for changes to ListArchives, then this (ListArchives is much closer to the dot/point):
```
The List Archives (Web & RTM) method has a new filter `events.types`
```
is easier for me "to process" than the current form:
```
There's a new `events.types` filter in the List Archives (Web & RTM) method.
```

Links to Web/RTM are inconsistent (ListArchives vs ListChats) 😉

**Comment by ale-kacperczyk** [comment]
@gluszczyk, I've applied your suggestions in the Chats section so that we can compare it visually.

**Comment by ale-kacperczyk** [comment]
> @gluszczyk, I've applied your suggestions in the Chats section so that we can compare it visually.

**Comment by ale-kacperczyk** [comment]
I wonder if now when the changes are grouped, perhaps we could reduce the number of links and only link the first occurrence of a method/push. The link would still be at hand, and we could gain more clarity.

**Comment by ale-kacperczyk** [comment]
Test

File: content/messaging/customer-chat-api/changelog/index.mdx [hunk 3]
```diff
@@ -11,51 +11,84 @@ This document is the record of all the changes in the **Customer Chat API**, Web
...
-### Added
+### Chats
+
+- The **Get Chats Summary** method was renamed to **List Chats** ([Web](/messaging/customer-chat-api/v3.2/#list-chats) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#list-chats)).
+- The **List Chats** ([Web](/messaging/customer-chat-api/v3.2/#list-chats) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#list-chats)) method:
+  - is paginated with `page_id`.
+  - had the `order` field replaced with `last_thread_created_at` in the response.
+- The **incoming_chat_thread** push was renamed to [**incoming_chat**](/messaging/customer-chat-api/v3.2/rtm-reference/#incoming_chat).
+- The **incoming_chat** push:
+  - had the `order` field in the `chat` object removed from the payload.
```
Comment (gluszczyk) [inline content/messaging/customer-chat-api/changelog/index.mdx:22]:
```
- had the `chat.order` field removed from the payload
```
wdyt?

File: content/messaging/customer-chat-api/changelog/index.mdx [hunk 4]
```diff
@@ -11,51 +11,84 @@ This document is the record of all the changes in the **Customer Chat API**, Web
...
+  - had the `user_type` field removed.
+- The **chat_user_removed** push was renamed to [**user_removed_from_chat**](/messaging/customer-chat-api/v3.2/rtm-reference/#user_removed_from_chat). 
+- The **user_removed_from_chat** push:
+  - has new fields: `reason` and `requester_id`.
+  - had the `user_type` field removed.
+
+### Events
+
+- The [**Form field**](/messaging/customer-chat-api/v3.2/#form-fields) data structure has a new field, `id` (`fields.answer.id` for the `group_chooser` field type). It's an identifier for each option Customers can choose. The `id` is now returned by the **Get Form** method ([Web](/messaging/customer-chat-api/v3.2/#get-form) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#get-form)) (`form.fields.options` for the `group_chooser` field type).
+- The  [**Thread**](/messaging/customer-chat-api/v3.2/#threads) data structure:
```
Comment (gluszczyk) [inline content/messaging/customer-chat-api/changelog/index.mdx:56]: double space 👀

File: content/messaging/customer-chat-api/changelog/index.mdx [hunk 5]
```diff
@@ -11,51 +11,84 @@ This document is the record of all the changes in the **Customer Chat API**, Web
...
 - The [**Property**](/messaging/customer-chat-api/v3.2/#properties) data structure now has the same format in requests as in responses and pushes.
-- There's a new field `id` in the [Form field](/messaging/customer-chat-api/v3.2/#form-fields) data structure (`fields.answer.id` for the `group_chooser` field type).  It's an identifier for each option Customers can choose. The `id` is now returned by the **Get Form** method ([Web](/messaging/customer-chat-api/v3.2/#get-form) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#get-form)) (`form.fields.options` for the `group_chooser` field type).
-- **Get URL Info** ([Web](/messaging/customer-chat-api/v3.2/#get-url-info) & [RTM](/messaging/customer-chat-api/v3.2/rtm-reference/#get-url-info)) now returns the `image_original_URL` parameter.
+- There are new methods:  **List License Properties** ([Web](/messaging/customer-chat-api/v3.2/#list-license-properties)) and **List Group Properties** ([Web](/messaging/customer-chat-api/v3.2/#list-group-properties)).
```
Comment (gluszczyk) [inline content/messaging/customer-chat-api/changelog/index.mdx:64]: double space :eyes: before `List License...`

File: content/messaging/customer-chat-api/changelog/index.mdx (hunk 3 above)
Comment (ale-kacperczyk) [reply content/messaging/customer-chat-api/changelog/index.mdx:22]: much better

**Pull request merged**